solution_template/jenkins/CreateUiDefinition.json

```diff
@@ -0,0 +1,193 @@
+        },
+        "elements": [
+          {
+            "name": "vmSize",
+            "type": "Microsoft.Compute.SizeSelector",
+            "label": "Size",
+            "toolTip": "",
+            "recommendedSizes": [
+              "Standard_DS2_v2",
+              "Standard_DS1_v2",
+              "Standard_DS11_v2"
+            ],
+            "osPlatform": "Linux",
+            "imageReference": {
+              "publisher": "Canonical",
+              "offer": "UbuntuServer",
+              "sku": "14.04.5-LTS"
+            },
+            "count": 1,
+            "visible": true
+          },
+          {
+            "name": "storage_type_hdd_true",
+            "type": "Microsoft.Common.TextBox",
+            "label": "",
+            "defaultValue": "Standard_LRS",
+            "toolTip": "",
+            "visible": false
+          },
+          {
+            "name": "storage_type_hdd_false",
+            "type": "Microsoft.Common.TextBox",
+            "label": "",
+            "defaultValue": "Premium_LRS",
+            "toolTip": "",
+            "visible": false
+          },
+          {
+            "name": "storageAccount",
+            "type": "Microsoft.Storage.StorageAccountSelector",
+            "label": "Storage account",
+            "toolTip": "Disks for Azure virtual machines are created in storage accounts",
+            "defaultValue": {
+              "name": "[concat('jenkins', take( toLower(encodeBase64(guid())) , 14), string(rand(0, 999)) )]",
+              "type": "[ first ( skip( parse('[\"Premium_LRS\", \"Standard_LRS\" ]'), int(basics('vmDiskType')) ) ) ]"
+            },
+            "options": {
+              "hideExisting": false
+            },
+            "constraints": {
+              "allowedTypes": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_RAGRS",
+                "Premium_LRS"
+              ]
+            },
+            "visible": true
+          },
+          {
+            "name": "publicIP",
+            "type": "Microsoft.Network.PublicIpAddressCombo",
+            "label": {
+              "publicIpAddress": "Public IP address",
+              "domainNameLabel": "Domain name label"
+            },
+            "toolTip": {
+              "publicIpAddress": "",
+              "domainNameLabel": ""
+            },
+            "defaultValue": {
+              "publicIpAddressName": "JenkinsIP",
+              "domainNameLabel": ""
+            },
+            "constraints": {
+              "required": {
+                "domainNameLabel": true
+              }
+            },
+            "options": {
+              "hideNone": true,
+              "hideDomainNameLabel": false,
+              "hideExisting": false
+            },
+            "visible": true
+          }
+        ]
+      }
+    ],
+    "outputs": {
+      "_artifactsLocation": "https://raw.githubusercontent.com/Azure/azure-devops-utils/v0.14.0",
```

Comment:
In the quickstarts we have a slash at the end of this url. Better safe than sorry I think